src/sagas.js

```diff
@@ -485,8 +483,8 @@ function* getConsoleVm (action) {
  * VmDetail is to be rendered.
  */
 export function* selectVmDetail (action) {
-  yield put(setVmDetailToShow({ vmId: action.payload.vmId }))
   yield fetchSingleVm(getSingleVm({ vmId: action.payload.vmId })) // async data refresh
+  yield put(loadInProgress({ value: false }))
```

Comment:
Please consider in context of https://github.com/oVirt/ovirt-web-ui/pull/198.

The loadInProgress is can be set even later on. Recently it's set to true:
- during app initialization (full data load is expected ...)
- when full reload is requested (the Refresh button)
- might be decided later to set it whenever polling occurs (recently is the user intentionally not informed about that)

For completeness, any VM action (or so) sets Action In Progress causing the same visual "Loading ..." component to render.

@bond95 , please review the use of "components/LoadingData" within this PR, there's no need to render it on every subscreen, I think.